lib/Bio/EnsEMBL/DataCheck/Checks/SeqRegionBRC4.pm

```diff
@@ -0,0 +1,171 @@
+=head1 LICENSE
+
+Copyright [2018-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::SeqRegionBRC4;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME           => 'SeqRegionBRC4',
+  DESCRIPTION    => 'Seq_region for BRC4 have correct attributes',
+  GROUPS         => ['brc4_core'],
+  DB_TYPES       => ['core'],
+  TABLES         => ['attrib_type', 'coord_system', 'external_db', 'seq_region', 'seq_region_attrib', 'seq_region_synonym']
+};
+
+sub tests {
+  my ($self) = @_;
+
+  my $species_id = $self->dba->species_id;
+
+  $self->check_seq_attrib_name($species_id, 'BRC4_seq_region_name');
+
+  # Check for INSDC accession (not systematic in case out assembly is not in sync with INSDC)
+  #$self->check_seq_synonym($species_id, 'INSDC');
```

Comment:
This could be moved to a separate 'advisory' datacheck. (But in MetaKeyBRC4 you have 'assemby.accession' as a required meta_key, which implies that the sequences must be in sync?)

Reply:
That's a problem for us when the assembly has an INSDC accession, but the sequences are actually different to what is currently in INSDC. This can happen when we have made updates to the assembly but haven't (yet) pushed the changes back to INSDC. I guess this should be an advisory check still, because this is not something we want to keep as is.